python/overlay_widget/shotgun_spinning_widget.py

```diff
@@ -0,0 +1,223 @@
+# Copyright (c) 2013 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+
+import math
+
+from tank.platform.qt import QtCore, QtGui
+
+# load resources
+from .ui import resources_rc # noqa
+
+
+class ShotgunSpinningWidget(QtGui.QWidget):
+    """
+    Overlay widget that can be placed on top over any QT widget.
```

Comment:
did you check the sphinx build?

feels like we need some examples at the very least!